### PR TITLE
spt_procss_state table added in the upgrade script.

### PR DESCRIPTION
**Pull request by lwinmoe**

### CODE REVIEW

File: src/pyasm/search/upgrade/project/config_upgrade.py [hunk 1]
```diff
@@ -19,6 +19,27 @@
...
+    def upgrade_v4_7_0_a01_001(self):
+        self.run_sql('''
+        CREATE TABLE "spt_process_state" (
+            "id" serial PRIMARY KEY,
+            "code" character varying(256),
+            "pipeline_code" character varying(256),
+            "process" character varying(256),
+            "search_type" character varying(256),
+            "search_code" character varying(256),
+            "timestamp" timestamp without time zone DEFAULT now() NOT NULL,
```
Comment (CeltonMcGrath) [inline src/pyasm/search/upgrade/project/config_upgrade.py]: timestamp without timezone may not exist in all database types

File: src/pyasm/search/upgrade/project/config_upgrade.py [hunk 2]
```diff
@@ -19,6 +19,27 @@
...
+            "status" jsonb,
```
Comment (CeltonMcGrath) [inline src/pyasm/search/upgrade/project/config_upgrade.py]: jsonb only exists in postgres

File: src/pyasm/search/upgrade/project/config_upgrade.py [hunk 3]
```diff
@@ -19,6 +19,27 @@
...
+            "state" character varying(256),
+            "data" jsonb,
```
Comment (CeltonMcGrath) [inline src/pyasm/search/upgrade/project/config_upgrade.py]: jsonb

File: VERSION [hunk 4]
```diff
@@ -1 +1 @@
-4.7.a01
\ No newline at end of file
+4.7.0.a01
```
Comment (CeltonMcGrath) [inline VERSION:1]: nice thank you

File: src/pyasm/search/upgrade/project/config_upgrade.py (hunk 1 above)
Comment (lwinmoe) [reply src/pyasm/search/upgrade/project/config_upgrade.py]: Added MySQL section with "timestamp DEFAULT CURRENT_TIMESTAMP NOT NULL"

File: src/pyasm/search/upgrade/project/config_upgrade.py (hunk 2 above)
Comment (lwinmoe) [reply src/pyasm/search/upgrade/project/config_upgrade.py]: json for MySQL added.

File: src/pyasm/search/upgrade/project/config_upgrade.py (hunk 3 above)
Comment (lwinmoe) [reply src/pyasm/search/upgrade/project/config_upgrade.py]: json for MySQL.

**Pull request merged**